webtau-core/src/test/groovy/com/twosigma/webtau/expectation/equality/handlers/HandlerMessagesTest.groovy

```diff
@@ -221,14 +221,14 @@ class HandlerMessagesTest {
     @Test
     void "equal stream and iterable"() {
         code {
-            actual(Stream.of(1)).should(equal(Arrays.asList(2)))
+            actual(Arrays.asList(1).stream()).should(equal(Arrays.asList(2)))
```

Comment:
[1].stream()?

Reply:
Stream.of(1)?

Reply:
Stream.of(1) is what was causing the problem in the first place!

Reply:
O boy :) clearly need more decaf green tea